src/Headers.php

```diff
@@ -74,7 +75,7 @@ public static function fromString($string)
 
             // check if a header name is present
             if (preg_match('/^(?P<name>[^()><@,;:\"\\/\[\]?={} \t]+):.*$/', $line, $matches)) {
-                if ($current) {
+                if (! empty($current)) {
```

Comment:
Avoid using `empty()`, since it ignores non-existing variables too. Consider `=== ''` instead